WasatchNET/Util.cs

```diff
@@ -246,6 +248,55 @@ public static byte[] truncateArray(byte[] src, int len)
             return tmp;
         }
 
+        public static float coeffConvertForward(float value, float[] coeffs)
+        {
+            float total = coeffs[0];
+
+            for (int i = 1; i < coeffs.Length; ++i)
+                total += (float)(coeffs[i] * Math.Pow(value, i));
+
+            return total;
+        }
+
+        public static float coeffConvertBackwardsNewton(float targetY, float[] coeffs, float minY, float maxY, float minX, float maxX)

```

Comment:
When would you use this?

Reply:
It's used for a "reverse calibration" without a second set of coeffs. E.g. I use it to find the mW for a given % of laser power (the calibration gives you a % to drive to a particular mW)